### PR TITLE
Addition of IGNORE rules for cases of pseudogenes linked to 3_prime |…

### PR DESCRIPTION
**Pull request by ens-LCampbell**
… 5_prime UTRs

Updates to valid_structures.conf to account for cases encountered in a MAKER2 annotation in which pseudogenes were defined with CDS, exon and UTRs. 

E.g:
[pseudo_with_UTRs.txt](https://github.com/Ensembl/ensembl-genomio/files/9072562/pseudo_with_UTRs.txt)

### CODE REVIEW

File: scripts/gff_metaparser/conf/valid_structures.conf [hunk 1]
```diff
@@ -150,6 +149,9 @@ gene/pseudogene/intron	IGNORE
 gene/trna/intron	IGNORE
 gene/sequence_variant	IGNORE
 
+## IGNORED STRANGE OR INVALID MODELS / PARTS
+pseudogene/three_prime_UTR	IGNORE:ignore_pseudogene_utr:UNSEEN
```
Comment (vsitnik) [inline scripts/gff_metaparser/conf/valid_structures.conf:153]: could you make them just [always] "IGNORE", under no flags ?

Comment (vsitnik) [reply scripts/gff_metaparser/conf/valid_structures.conf:153]: or do you prefer to have a flag to report them by default?

Comment (ens-LCampbell) [reply scripts/gff_metaparser/conf/valid_structures.conf:153]: If we set them to IGNORE outright, they are not going to be reported in the gff_stats.err log file right ? I think let's leave it for now. Its not something seen very often and reporting it then setting the flag in conf isn't difficult to do.

Comment (vsitnik) [reply scripts/gff_metaparser/conf/valid_structures.conf:153]: makes sense. thanks!

**Review by vsitnik — APPROVED** [review]
ok

**Pull request merged**